src/modules/item/selectors.ts

```diff
@@ -94,35 +96,57 @@ export const getEntityByItemId = createSelector<RootState, Entity[], Record<stri
     }, {} as Record<string, Entity>)
 )
 
+const getItemSyncedStatus = (item: Item, entity: Entity | null) => {
+  let status = SyncStatus.UNSYNCED
+  if (!entity) {
+    status = SyncStatus.LOADING
+  } else if (areSynced(item, entity)) {
+    status = SyncStatus.SYNCED
+  }
+  return status
+}
+
+const getStatusForTPW = (item: Item, itemCuration: ItemCuration | null, entity: Entity): SyncStatus => {
```

Comment:
Nitpick but can we drop the W? `getStatusForTP`

this is probably not the only place where we use TPW but it's kinda wrong as the W stands for Wearable and it's a bit weird when we're managing Collections and Items. Third Party (TP) seems a bit better.

This is a mea-culpa as we started this mess with Lautaro lol

Reply:
Glad you asked that 😂 .`W` dropped!